a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -178,13 +178,14 @@ class ConfigureVirtEthIfaces(BaseVThunderTask):
     def execute(self, ve_interfaces, loadbalancer, vthunder):
         """Execute to configure ve on thunder device."""
         amphora_id = loadbalancer.amphorae[0].id
-        ve_segments = ve_interfaces[amphora_id]
-        c = self.client_factory(vthunder)
-        ve_client = acos_client.v30.interface.VirtualEthernet(c)
-        for segment, fixed_ip in six.iteritems(ve_segments):
-            netmask = '/' + fixed_ip.subnet.cidr.split('/')[1]
-            ve_client.update(segment, ip_address=fixed_ip.ip_address,
-                             ip_netmask=netmask)
+        if ve_interfaces:
+	    ve_segments = ve_interfaces[amphora_id]
```

Comment:
Indentation here is wrong

Reply:
This is fine. It is tested.

![image](https://user-images.githubusercontent.com/52992745/74910029-93c20580-53df-11ea-9b29-0a28323b2af4.png)